### PR TITLE
Fix bad project slug in the publishing scripts, plus fix a typo.

### PR DESCRIPTION
**Pull request by cgruber**
Noted by @sameb in #196 after merging. Woops.

### CODE REVIEW

File: util/generate-latest-docs.sh [hunk 1]
```diff
@@ -1,6 +1,6 @@
 # see http://benlimmer.com/2013/12/26/automatically-publish-javadoc-to-gh-pages-with-travis-ci/ for details
 
-if [ "$TRAVIS_REPO_SLUG" == "google/truth" ] && \
+if [ "$TRAVIS_REPO_SLUG" == "google/auto" ] && \
    [ "$TRAVIS_JDK_VERSION" == "oraclejdk7" ] && \
    [ "$TRAVIS_PULL_REQUEST" == "false" ] && \
    [ "$TRAVIS_BRANCH" == "master" ]; then
```
Comment (eamonnmcmanus) [inline util/generate-latest-docs.sh]: Still seeing a reference to truth (on the git clone line).

Comment (cgruber) [inline util/generate-latest-docs.sh]: I'm confused. what do you mean "on the git clone?"

Comment (cgruber) [inline util/generate-latest-docs.sh]: Oh... dammit.

Comment (cgruber) [inline util/generate-latest-docs.sh]: fixed

**Pull request merged**